packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -752,7 +880,7 @@ describe('FLE2', function () {
   let browser: CompassBrowser;
 
   before(async function () {
-    skipForWeb(this, 'connect form not available in compass-web');
+    skipForWeb(this, 'FLE2 not available in compass-web');
```

Comment:
```suggestion
    skipForWeb(this, 'QE not available in compass-web');
```

(FLE2 is no longer a name we use) or

```suggestion
    skipForWeb(this, 'client-side encryption not available in compass-web');
```

(to cover both CSFLE/QE)